packages/demo-legacy/yarn.lock

```diff
@@ -2,61 +2,41 @@
 # yarn lockfile v1
```

Comment:
this says:
`packages/child-app/yarn.lock → packages/demo-legacy/yarn.lock `
what happened here? was the file deleted?